### PR TITLE
OCP4 STIG: SRGs related to NIST CM-11 are inherently met

### PR DESCRIPTION
**Pull request by jhrozek**
#### Description:

- Adds justification and artifact description to two SRGs that handle SRGs related
  to CM-11

#### Rationale:

- OCP4 STIG

#### Review Hints:

- I will post a link to an SRG export later

### CODE REVIEW

**Comment by jhrozek** [comment]
To review, compare the [SRG mapping](https://github.com/ComplianceAsCode/content/suites/11225359251/artifacts/574510282) generated by the GH actions with the [current version](https://docs.google.com/spreadsheets/d/1ouRalvXzRHuhPPN_AKMvqqscUqGvVOkkEdMP3YYBO44/edit#gid=1859516253) of the DISA STIG.

**Comment by jhrozek** [comment]
@Vincent056 @rhmdnd PTAL

File: controls/srg_ctr/SRG-APP-000378-CTR-000885.yml [hunk 1]
```diff
@@ -5,3 +5,15 @@ controls:
...
+    https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html
+  status_justification: |-
+    User's of the OpenShift platform do not directly instantiate
+    a container image. Instead in order to instantiate a container, the user creates
+    a Deployment object with the details of the requested state.  OpenShift will then
+    instantiate the containers inside pods on behalf of the users using a service
+    account. OpenShift employs a robust RBAC policy to allow an administrator to control
+    who has access to create, update, modify container deployments. Thus only user's
+    with the appropriate permissions to a given namespace may create or modify deployment
+    objects.\n \nhttps://docs.openshift.com/container-platform/latest/authentication/using-rbac.html
```
Comment (Vincent056) [inline controls/srg_ctr/SRG-APP-000378-CTR-000885.yml:19]: This might be nit-picking, are we supposed to remove the escape new line `\n` here

**Review by Vincent056 — APPROVED** [review]
/lgtm
thanks for addressing it

**Pull request merged**